aspects/schema.go

```diff
@@ -444,3 +446,86 @@ func (v *stringSchema) parseConstraints(constraints map[string]json.RawMessage)
 
 	return nil
 }
+
+type intSchema struct {
+	min     *int
```

Comment:
Just to double check, do we want to use a platform dependant type here? Seems this might have portability implications

Reply:
The spec doesn't mention bounds to integers so maybe it's better to confirm this with @pedronis

Reply:
as we use DecodeWithNumber I think as we should? I think these should use int64 ?

Reply:
I'll change the unmarshalling to use Number and these to int64

Reply:
Changed to 64bit ints, thanks for raising this Ernest